system/Encryption/Encryption.php

```diff
@@ -149,7 +149,7 @@ public static function createKey($length = 32)
      *
      * @param string $key Property name
      *
-     * @return mixed
+     * @return array|EncrypterInterface|string|null
```

Comment:
This method returns only 'key', 'digest', 'driver', 'drivers'. See `__isset()`.
```suggestion
     * @return array|string|null
```

Reply:
Done.